crates/red_knot_python_semantic/resources/mdtest/import/basic.md

```diff
@@ -116,9 +116,15 @@ reveal_type(c.C)  # revealed: Literal[C]
 class C: ...
 ```
 
+## Unresolvable module import
+
+```py snapshot-diagnostic
+import zqzqzqzqzqzqzq
+```
+
 ## Unresolvable submodule imports
 
-```py
+```py snapshot-diagnostic
```

Comment:
Depending how we resolve my other comment about inline assertions vs snapshots, and how we intend to use snapshotting, I might prefer to have a dedicated file demonstrating snapshotting, rather than a couple random tests in the import semantics test.

If these stay here, then I don't want to omit the inline assertion comment on the "Unresolvable module import" test.

Reply:
I ended up rolling back the change that ignores unmatched diagnostics when there aren't any inline assertions. And I moved this config to be a section-level config.